File: src/main/java/facebook/yoga/event/MeasureCallbackEndEventData.java
package facebook.yoga.event;

//C++ TO JAVA CONVERTER WARNING: The original type declaration contained unconverted modifiers:
//ORIGINAL LINE: struct Event::TypedData<Event::MeasureCallbackEnd>
//C++ TO JAVA CONVERTER TODO TASK: C++ template specialization was removed by C++ to Java Converter:
public class MeasureCallbackEndEventData //Type originates from: event.h
{
  public Object layoutContext;
  public float width;
  public YGMeasureMode widthMeasureMode;
  public float height;
  public YGMeasureMode heightMeasureMode;
  public float measuredWidth;
  public float measuredHeight;
  public facebook.yoga.LayoutPassReason reason;
}

File: src/commonMain/kotlin/com/joingo/yoga/internal/event/MeasureCallbackEndEventData.kt
package com.joingo.yoga.internal.event

import com.joingo.yoga.internal.enums.YGMeasureMode

class MeasureCallbackEndEventData(
    val layoutContext: Any?,
    val width: Float,
    val widthMeasureMode: YGMeasureMode,
    val height: Float,
    val heightMeasureMode: YGMeasureMode,
    val measuredWidth: Float,
    val measuredHeight: Float,
    val reason: LayoutPassReason
) : CallableEvent() //Type originates from: event.h
